--- samples/logging_proxy.c	2012-10-16 22:02:02.000000000 +0200
+++ samples/logging_proxy.c	2012-10-16 22:02:36.000000000 +0200
@@ -25,7 +25,6 @@
 #include <unistd.h>

 #include <glib.h>
-#include <glib/glist.h>

 #include <mw_cipher.h>
 #include <mw_common.h>
--- samples/login_server.c	2012-10-16 22:02:02.000000000 +0200
+++ samples/login_server.c	2012-10-16 22:02:23.000000000 +0200
@@ -19,7 +19,6 @@
 #include <unistd.h>

 #include <glib.h>
-#include <glib/glist.h>

 #include <mw_cipher.h>
 #include <mw_common.h>
--- samples/nocipher_proxy.c	2012-10-16 22:02:02.000000000 +0200
+++ samples/nocipher_proxy.c	2012-10-16 22:02:56.000000000 +0200
@@ -29,7 +29,6 @@
 #include <unistd.h>

 #include <glib.h>
-#include <glib/glist.h>

 #include <mw_common.h>
 #include <mw_message.h>
--- samples/redirect_server.c	2012-10-16 22:02:02.000000000 +0200
+++ samples/redirect_server.c	2012-10-16 22:02:48.000000000 +0200
@@ -22,7 +22,6 @@
 #include <unistd.h>

 #include <glib.h>
-#include <glib/glist.h>

 #include <mw_common.h>
 #include <mw_message.h>
--- src/channel.c	2012-10-16 22:02:02.000000000 +0200
+++ src/channel.c	2012-10-16 22:03:01.000000000 +0200
@@ -19,8 +19,6 @@
 */

 #include <glib.h>
-#include <glib/ghash.h>
-#include <glib/glist.h>
 #include <string.h>

 #include "mw_channel.h"
--- src/mw_debug.c	2012-10-16 22:02:02.000000000 +0200
+++ src/mw_debug.c	2012-10-16 22:03:06.000000000 +0200
@@ -19,7 +19,7 @@
 */


-#include <glib/gstring.h>
+#include <glib.h>

 #include "mw_debug.h"

--- src/mw_message.h	2012-10-16 22:02:02.000000000 +0200
+++ src/mw_message.h	2012-10-16 22:03:09.000000000 +0200
@@ -22,7 +22,7 @@
 #define _MW_MESSAGE_H


-#include <glib/glist.h>
+#include <glib.h>
 #include "mw_common.h"


--- src/mw_srvc_conf.h	2012-10-16 22:02:02.000000000 +0200
+++ src/mw_srvc_conf.h	2012-10-16 22:03:17.000000000 +0200
@@ -22,7 +22,7 @@
 #define _MW_SRVC_CONF_H


-#include <glib/glist.h>
+#include <glib.h>
 #include "mw_common.h"


--- src/mw_srvc_dir.h	2012-10-16 22:02:02.000000000 +0200
+++ src/mw_srvc_dir.h	2012-10-16 22:03:35.000000000 +0200
@@ -22,7 +22,6 @@


 #include <glib.h>
-#include <glib/glist.h>


 #ifdef __cplusplus
--- src/mw_srvc_place.h	2012-10-16 22:02:02.000000000 +0200
+++ src/mw_srvc_place.h	2012-10-16 22:03:40.000000000 +0200
@@ -22,7 +22,7 @@
 #define _MW_SRVC_PLACE_H


-#include <glib/glist.h>
+#include <glib.h>
 #include "mw_common.h"


--- src/mw_srvc_resolve.h	2012-10-16 22:02:02.000000000 +0200
+++ src/mw_srvc_resolve.h	2012-10-16 22:03:44.000000000 +0200
@@ -23,7 +23,6 @@


 #include <glib.h>
-#include <glib/glist.h>


 #ifdef __cplusplus
--- src/mw_st_list.h	2012-10-16 22:02:02.000000000 +0200
+++ src/mw_st_list.h	2012-10-16 22:03:48.000000000 +0200
@@ -30,7 +30,6 @@


 #include <glib.h>
-#include <glib/glist.h>
 #include "mw_common.h"


--- src/mw_util.h	2012-10-16 22:02:02.000000000 +0200
+++ src/mw_util.h	2012-10-16 22:03:51.000000000 +0200
@@ -23,8 +23,6 @@


 #include <glib.h>
-#include <glib/ghash.h>
-#include <glib/glist.h>


 #define map_guint_new() \
--- src/srvc_aware.c	2012-10-16 22:02:02.000000000 +0200
+++ src/srvc_aware.c	2012-10-16 22:03:58.000000000 +0200
@@ -19,8 +19,6 @@
 */

 #include <glib.h>
-#include <glib/ghash.h>
-#include <glib/glist.h>
 #include <string.h>

 #include "mw_channel.h"
--- src/srvc_conf.c	2012-10-16 22:02:02.000000000 +0200
+++ src/srvc_conf.c	2012-10-16 22:04:01.000000000 +0200
@@ -19,8 +19,6 @@
 */

 #include <glib.h>
-#include <glib/ghash.h>
-#include <glib/glist.h>

 #include <stdio.h>
 #include <stdlib.h>
--- src/srvc_dir.c	2012-10-16 22:02:02.000000000 +0200
+++ src/srvc_dir.c	2012-10-16 22:04:07.000000000 +0200
@@ -18,7 +18,7 @@
   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 */

-#include <glib/ghash.h>
+#include <glib.h>

 #include "mw_channel.h"
 #include "mw_common.h"
--- src/srvc_ft.c	2012-10-16 22:02:02.000000000 +0200
+++ src/srvc_ft.c	2012-10-16 22:04:12.000000000 +0200
@@ -19,7 +19,7 @@
 */


-#include <glib/glist.h>
+#include <glib.h>

 #include "mw_channel.h"
 #include "mw_common.h"
--- src/srvc_im.c	2012-10-16 22:02:02.000000000 +0200
+++ src/srvc_im.c	2012-10-16 22:04:18.000000000 +0200
@@ -19,7 +19,6 @@
 */

 #include <glib.h>
-#include <glib/glist.h>
 #include <string.h>

 #include "mw_channel.h"
--- src/srvc_place.c	2012-10-16 22:02:02.000000000 +0200
+++ src/srvc_place.c	2012-10-16 22:04:23.000000000 +0200
@@ -19,8 +19,6 @@
 */

 #include <glib.h>
-#include <glib/ghash.h>
-#include <glib/glist.h>

 #include <stdio.h>
 #include <stdlib.h>
--- src/srvc_resolve.c	2012-10-16 22:02:02.000000000 +0200
+++ src/srvc_resolve.c	2012-10-16 22:04:27.000000000 +0200
@@ -18,7 +18,7 @@
   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 */

-#include <glib/ghash.h>
+#include <glib.h>

 #include "mw_channel.h"
 #include "mw_common.h"
--- src/srvc_store.c	2012-10-16 22:02:02.000000000 +0200
+++ src/srvc_store.c	2012-10-16 22:04:31.000000000 +0200
@@ -18,7 +18,7 @@
   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 */

-#include <glib/glist.h>
+#include <glib.h>

 #include "mw_channel.h"
 #include "mw_debug.h"
--- src/st_list.c	2012-10-16 22:02:02.000000000 +0200
+++ src/st_list.c	2012-10-16 22:04:36.000000000 +0200
@@ -20,7 +20,7 @@

 #include <stdio.h>
 #include <string.h>
-#include <glib/gstring.h>
+#include <glib.h>

 #include "mw_debug.h"
 #include "mw_util.h"
