--- src/session.c 2005-12-15 14:11:47.000000000 -0600
+++ src/session.c 2012-08-22 08:40:54.043701375 -0500
@@ -448,6 +448,8 @@
   property_set(s, mwSession_SERVER_VER_MAJOR, GPOINTER(msg->major), NULL);
   property_set(s, mwSession_SERVER_VER_MINOR, GPOINTER(msg->minor), NULL);
 
+ g_info("Sametime server version (major/minor): 0x%x / 0x%x", msg->major, msg->minor);
+
   /* compose the login message */
   log = (struct mwMsgLogin *) mwMessage_new(mwMessage_LOGIN);
   log->login_type = GUINT(property_get(s, mwSession_CLIENT_TYPE_ID));