--- cyrus-sasl-2.1.21/lib/client.c	2004-11-17 08:19:30.000000000 -0600
+++ cyrus-sasl-2.1.21-mine/lib/client.c	2007-10-22 21:44:06.000000000 -0500
@@ -59,9 +59,9 @@
 #include "saslutil.h"
 #include "saslint.h"
 
-static cmech_list_t *cmechlist; /* global var which holds the list */
+extern sasl_global_callbacks_t global_callbacks;
 
-static sasl_global_callbacks_t global_callbacks;
+static cmech_list_t *cmechlist; /* global var which holds the list */
 
 static int _sasl_client_active = 0;
 
--- cyrus-sasl-2.1.21/plugins/gssapi.c	2004-07-21 09:39:06.000000000 -0500
+++ cyrus-sasl-2.1.21-mine/plugins/gssapi.c	2007-10-22 21:34:05.000000000 -0500
@@ -46,7 +46,7 @@
 #include <config.h>
 
 #ifdef HAVE_GSSAPI_H
-#include <gssapi.h>
+#include <gssapi/gssapi.h>
 #else
 #include <gssapi/gssapi.h>
 #endif

--- cyrus-sasl-2.1.21/plugins/kerberos4.c	2005-01-10 01:08:53.000000000 -0600
+++ cyrus-sasl-2.1.21-mine/plugins/kerberos4.c	2007-10-22 21:34:05.000000000 -0500
@@ -50,7 +50,8 @@
 
 #ifdef WITH_DES
 # ifdef WITH_SSL_DES
-#  include <openssl/des.h>
+//#  include <openssl/des.h>
+#  include <des.h>
 # else
 #  include <des.h>
 # endif /* WITH_SSL_DES */

--- cyrus-sasl-2.1.21/Makefile.in	2007-10-23 12:32:17.000000000 -0500
+++ cyrus-sasl-2.1.21/Makefile2.in	2007-10-23 12:32:03.000000000 -0500
@@ -35,6 +35,7 @@
 NORMAL_UNINSTALL = :
 PRE_UNINSTALL = :
 POST_UNINSTALL = :
+build_triplet = @build@
 host_triplet = @host@
 ACLOCAL = @ACLOCAL@
 AMDEP_FALSE = @AMDEP_FALSE@

--- cyrus-sasl-2.1.21/include/Makefile.in	2007-10-23 12:32:17.000000000 -0500
+++ cyrus-sasl-2.1.21/include/Makefile2.in	2007-10-23 12:32:03.000000000 -0500
@@ -35,6 +35,7 @@
 NORMAL_UNINSTALL = :
 PRE_UNINSTALL = :
 POST_UNINSTALL = :
+build_triplet = @build@
 host_triplet = @host@
 ACLOCAL = @ACLOCAL@
 AMDEP_FALSE = @AMDEP_FALSE@

--- cyrus-sasl-2.1.21/lib/Makefile.in	2007-10-23 12:32:17.000000000 -0500
+++ cyrus-sasl-2.1.21/lib/Makefile2.in	2007-10-23 12:32:03.000000000 -0500
@@ -35,6 +35,7 @@
 NORMAL_UNINSTALL = :
 PRE_UNINSTALL = :
 POST_UNINSTALL = :
+build_triplet = @build@
 host_triplet = @host@
 ACLOCAL = @ACLOCAL@
 AMDEP_FALSE = @AMDEP_FALSE@

--- cyrus-sasl-2.1.21/plugins/Makefile.in	2007-10-23 12:32:17.000000000 -0500
+++ cyrus-sasl-2.1.21/plugins/Makefile2.in	2007-10-23 12:32:03.000000000 -0500
@@ -35,6 +35,7 @@
 NORMAL_UNINSTALL = :
 PRE_UNINSTALL = :
 POST_UNINSTALL = :
+build_triplet = @build@
 host_triplet = @host@
 ACLOCAL = @ACLOCAL@
 AMDEP_FALSE = @AMDEP_FALSE@

--- cyrus-sasl-2.1.21/pwcheck/Makefile.in	2007-10-23 12:32:17.000000000 -0500
+++ cyrus-sasl-2.1.21/pwcheck/Makefile2.in	2007-10-23 12:32:03.000000000 -0500
@@ -35,6 +35,7 @@
 NORMAL_UNINSTALL = :
 PRE_UNINSTALL = :
 POST_UNINSTALL = :
+build_triplet = @build@
 host_triplet = @host@
 ACLOCAL = @ACLOCAL@
 AMDEP_FALSE = @AMDEP_FALSE@


--- cyrus-sasl-2.1.21/sample/Makefile.in	2007-10-23 12:32:17.000000000 -0500
+++ cyrus-sasl-2.1.21/sample/Makefile2.in	2007-10-23 12:32:03.000000000 -0500
@@ -35,6 +35,7 @@
 NORMAL_UNINSTALL = :
 PRE_UNINSTALL = :
 POST_UNINSTALL = :
+build_triplet = @build@
 host_triplet = @host@
 ACLOCAL = @ACLOCAL@
 AMDEP_FALSE = @AMDEP_FALSE@


--- cyrus-sasl-2.1.21/saslauthd/Makefile.in	2007-10-23 12:32:17.000000000 -0500
+++ cyrus-sasl-2.1.21/saslauthd/Makefile2.in	2007-10-23 12:32:03.000000000 -0500
@@ -35,6 +35,7 @@
 NORMAL_UNINSTALL = :
 PRE_UNINSTALL = :
 POST_UNINSTALL = :
+build_triplet = @build@
 host_triplet = @host@
 ACLOCAL = @ACLOCAL@
 AMDEP_FALSE = @AMDEP_FALSE@


--- cyrus-sasl-2.1.21/sasldb/Makefile.in	2007-10-23 12:32:17.000000000 -0500
+++ cyrus-sasl-2.1.21/sasldb/Makefile2.in	2007-10-23 12:32:03.000000000 -0500
@@ -35,6 +35,7 @@
 NORMAL_UNINSTALL = :
 PRE_UNINSTALL = :
 POST_UNINSTALL = :
+build_triplet = @build@
 host_triplet = @host@
 ACLOCAL = @ACLOCAL@
 AMDEP_FALSE = @AMDEP_FALSE@


--- cyrus-sasl-2.1.21/utils/Makefile.in	2007-10-23 12:32:17.000000000 -0500
+++ cyrus-sasl-2.1.21/utils/Makefile2.in	2007-10-23 12:32:03.000000000 -0500
@@ -35,6 +35,7 @@
 NORMAL_UNINSTALL = :
 PRE_UNINSTALL = :
 POST_UNINSTALL = :
+build_triplet = @build@
 host_triplet = @host@
 ACLOCAL = @ACLOCAL@
 AMDEP_FALSE = @AMDEP_FALSE@

